VERSION

```diff
@@ -1 +1 @@
-3.10.2
+3.9.4
```

Comment:
Why downgrade?

Reply:
b/c i manually pushed 3.10.2 🤦🏻 and needed to verify github action can push it